build/teamcity/cockroach/nightlies/roachtest_nightly_impl.sh

```diff
@@ -62,6 +62,7 @@ build/teamcity-roachtest-invoke.sh \
   --select-probability="${select_probability}" \
   --cloud="${CLOUD}" \
   --count="${COUNT-1}" \
+  --auto-kill-threshold="${AUTO_KILL_THRESHOLD:-0.15}" \
```

Comment:
I know the issue linked to this PR does mention 15%, but I wanted to discuss this again with everyone: do we think this is a good default?

Looking at the latest nightly on GCE, I see we have 514 tests in a run so, if we use the 15% value, it would take _77_ failures before the run is aborted.

In comparison, the "bad" nightly run that motivated this issue (that was canceled manually), had _36_ test failures. It seems to me that we should be canceling builds a little more proactively.

Reply:
So something around 7% as the threshold?

Reply:
I'd go even lower to 5%, which should still be rather conservative. The mean is somewhere below 3% and std. deviation isn't that high either.

Reply:
Set to 5%